lib/test-definition.rb

```diff
@@ -331,6 +331,14 @@ def skip_unless_icscf
     raise SkipThisTest.new "No I-CSCF in deployment", "Call with ICSCF_HOSTNAME=<hostname>" unless ENV['ICSCF_HOSTNAME']
   end
 
+  def skip_unless_emergency_reg_support
+    raise SkipThisTest.new "Not running with emergency registrations", "Call with EMERGENCY_REG=true to run test" unless ENV['EMERGENCY_REG']
```

Comment:
No biggie, but you recommend `EMERGENCY_REG=true` here and `EMERGENCY_REG=Y` in the README

Reply:
@sebrexmetaswitch For consistency with the same consistent inconsistency elsewhere :-)   Really I should have fixed to make it consistently consistent everywhere!

Reply:
Actually -- there was only one previous instance.  Fixed.